src/main/java/it/gov/pagopa/pu/send/model/SendNotification.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.send.model;
+
+import it.gov.pagopa.pu.send.enums.NotificationStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldNameConstants;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Document("send_notification")
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@FieldNameConstants
+public class SendNotification {
+
+  @Transient
+  public static final String SEQUENCE_NAME = "send_notification_sequence";
+
+  @Id
+  private String id;
+  private Long sendNotificationId;
```

Comment:
these fields are ambiguous.
If I see SendNotification.getSendNotificationId() I would expect it to be the @Id of the collection.
What about just `notificationId`, while naming id into `sendNotificationId` in order to have same naming convention of the other application?
```suggestion
  private String SendNotificationId;
  private Long notificationId;
```